lib/Tree.js

```diff
@@ -599,7 +599,7 @@ Tree.prototype.predict = function (inputData, path, missingStrategy, median,
     'path': path,
     'confidence': this.confidence,
     'distribution': this.distribution,
-    'count': getInstances(this.distribution),
+    'count': this.count,
```

Comment:
Just a quick check: it seems `getInstances` is not used anywhere in the project (anymore). So, how does the `this.count` property get initialised? I was expecting some kind of equivalent logic somewhere and it is not apparent to me what replaces `getInstances`... (but maybe I should look better). BTW, if it is correct that `getInstances` is not used anymore, maybe it is a good candidate for cleanup.

Reply:
We've replaced getInstances with the count of instances in the node. The side effec is that weighted models will use this count to compute the associated confidence. You're right about the fact that getInstances is not used then anywhere, so we can get rid of it.

Reply:
ok, thanks -- so, the count now comes from the resource JSON, I understand.

Reply:
yes, it's the original count of instances in the node